config/crd/bases/konk.infoblox.com_konks.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: konks.konk.infoblox.com
```

Comment:
konks.konk makes me sad. Do you think this is because the chart and group are named konk?

Reply:
Yes, it is. You can see it in the separate commit I made to change the group: https://github.com/infobloxopen/konk/pull/9/commits/4f1c3df290fbcf1ab688f9dfec2c7a4b87c27e9b#diff-864ecbfdcb97f08d23574c2028bad92eR5